app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -36,17 +42,28 @@ class ExportWorkflowsDialog extends React.Component {
     getWorkflowsInOrder(project, { fields: 'display_name' })
       .then((workflows) => {
         this.setState({ workflows, loading: false });
-      });
-  }
 
-  toggleExport() {
-    if (this.workflowList.selectedIndex >= 0) {
-      this.setState({ workflowSelected: true });
-    }
+        workflows.forEach((workflow) => {
+          workflow.get('classifications_export')
+            .then((media) => {
+              const mediaState = this.state.media;
+              mediaState[workflow.id] = media[0];
+              this.setState({ media: mediaState });
+            })
+            .catch((error) => {
+              if (error.status === 404) {
```

Comment:
I think this is the reverse of what we were chatting about. The 404 just indicates the project owner hasn't requested this export before, so we don't really care about reporting that back to the user. What I think this should be is:
```
if (error.status !== 404) {
  this.props.onFail.(error);
}
```